test/test_acoustid_tagger.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from headphones2.postprocess.taggers.acoustid_tagger import AcoustIDAlbumTagger
+from headphones2.postprocess.process_folder import FolderIterator
+from beets.library import Item
+from conftest import vcr
+
+HUMAN_EQUATION_MBID = '90392b89-77fc-3c39-813d-c5c09abf8943'
+DARK_MEDICINE_ACOUSTID = 'f7c6418c-d8a7-46f8-bbeb-d5a11481776e'
+
+@pytest.fixture
+def single_item():
+    track_22 = r"C:\Users\Omer\Desktop\The Theory of Everything [2013]\20 Ayreon - The Theory of Everything [2013] - Phase 2_ Symmetry, Dark Medicine.flac"
+    item = Item.from_path(str(track_22))
+    item.acoustid_id = ''
+    item.acoustid_fingerprint = ''
+    item.write()
+    return item
+
+
+@pytest.fixture
+def album_item_list():
+    folder = "M:\Ayreon\The Human Equation [2004]"
+    items = [Item.from_path(str(p)) for p in FolderIterator(folder, extensions=['.mp3', '.flac'])]
+    for item in items:
+        item.acoustid_id = ''
+        item.acoustid_fingerprint = ''
+        item.write()
```

Comment:
If you change the code so that after the `process` we do a `write` on each item.
It will move the IO to the top of the code, it will be easier to test (Create an item without the ids, check after the process that they are added. never write them to disk).
See http://www.youtube.com/watch?v=DJtef410XaM.